dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/exchange/Request.java

```diff
@@ -99,6 +99,10 @@ public void setEvent(String event) {
         mData = event;
     }
 
+    public void setEvent(boolean mEvent) {
```

Comment:
looks like we should remove

```java
   public void setEvent(String event) {
        mEvent = true;
        mData = event;
    }
```

Reply:
now I see why the original setEvent() is necessary, pls. keep it. Sorry about this.

Reply:
Setting only the mEvent indicate that there an event but it does tell what kind of event it was. So should we set just set single value? I beleive setting only boolean may not provide complete information what do you say?

Reply:
> Setting only the mEvent indicate that there an event but it does tell what kind of event it was. So should we set just set single value? I beleive setting only boolean may not provide complete information what do you say?

Because mData will carry the relevant value, so here is simply to mark whether the request is an event